console/src/assets/i18n/en.json

```diff
@@ -1837,6 +1841,12 @@
       "UPLOADPRIVATEKEY": "Upload Private Key",
       "KEYMAXSIZEEXCEEDED": "Maximum size of 5kB exceeded."
     },
+    "SAML": {
+      "METADATAXML": "Metadata Xml",
+      "METADATAURL": "Metadata URL",
+      "BINDING": "Binding",
+      "SIGNEDREQUEST": "Signed Request"
+    },
```

Comment:
Please translate to other languages or give us access to edit the PR :slightly_smiling_face: